lib/desktop/commands/list.rb

```diff
@@ -79,6 +80,7 @@ def run
       def session_to_array(s)
         if s.state == :broken
           [
+            s.name,
```

Comment:
This branch is used when loading the `metadata.yml` file failed for some reason.  If we get here, we do we know that it's safe to call `s.name`?  It might be.

Could you add a syntax error to `metadata.yml` and test?  Adding the following line to the end of the file should do that:

```
foo: [
```